### DISCUSSION TITLE
max length Inspector

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2331] [Q&A]
### Introduction

is there any option to provide max and min length for the inspector input field 

![image](https://github.com/clientIO/joint/assets/117301232/69e4edf0-ac72-4af4-9a7a-aac66afaf7ed)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
It's possible with [`<input type="text">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/text) and [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement).
```
attribute: {
    type: 'text',
    attrs: {
        input: {
            minlength: 4,
            maxlength: 8
        }
    }
},
```
